_inc/client/my-plan/my-plan-header/index.js

```diff
@@ -146,6 +164,54 @@ class MyPlanHeader extends React.Component {
 				);
 				break;
 
+			case 'is-daily-backup-plan':
+				planCard = (
+					<PlanHeaderCard
+						title={ __( 'Your plan: Jetpack Free + Daily Backup' ) }
+						text={ __(
+							'Worried about security? Get automated security fixes and more: {{a}}Upgrade now{{/a}}',
+							{
+								components: {
+									a: (
+										<UpgradeLink
+											source="my-plan-header-daily-backup-text-link"
+											target="upgrade-now"
+											feature="my-plan-header-daily-backup-upgrade"
+										/>
+									),
+								},
+							}
+						) }
+						imgSrc={ imagePath + '/plans/plan-free.svg' }
+						imgAlt={ __( 'Jetpack Daily Backup Plan' ) }
+					/>
+				);
+				break;
+
+			case 'is-realtime-backup-plan':
+				planCard = (
+					<PlanHeaderCard
+						title={ __( 'Your plan: Jetpack Free + Real-time Backup' ) }
+						text={ __(
+							'Worried about security? Get automated security fixes and more: {{a}}Upgrade now{{/a}}',
+							{
+								components: {
+									a: (
+										<UpgradeLink
+											source="my-plan-header-real-time-backup-text-link"
+											target="upgrade-now"
+											feature="my-plan-header-real-time-backup-upgrade"
+										/>
+									),
+								},
+							}
+						) }
+						imgSrc={ imagePath + '/plans/plan-free.svg' }
```

Comment:
We currently change this icon when you switch to a different plan. Should we change it for those 2 new plans as well?

![image](https://user-images.githubusercontent.com/426388/67009337-b7043d80-f0eb-11e9-825b-fa59317dd80a.png)

Reply:
The Figma designs for this have it using the same icon so that is what I went with. I'll ask for clarification to be sure though and update in a later PR if needed.